plan/logical_plan_builder.go

```diff
@@ -902,7 +909,11 @@ func (a *havingAndOrderbyExprResolver) Leave(n ast.Node) (node ast.Node, ok bool
 					return n, true
 				}
 			}
-			a.err = errors.Errorf("Unknown Column %s", v.Name.Name.L)
+			cn := v.Name.Name.L
```

Comment:
It seems this logic will be used in many places. Can we make it a function ?

Reply:
I didn't really find another place use this logic totally the same.
May keep it?